oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/CompressedDocumentPropertyStateTest.java

```diff
@@ -238,6 +238,11 @@ private void getBrokenSurrogateAndInitializeDifferentStores(DocumentStoreFixture
             if (nodeStore != null) {
                 nodeStore.dispose();
             }
+            try {
```

Comment:
In theory, disposing the node store should take care of this. We should find out why it doesn't.

Reply:
It is just a precocious adding. No truly impact.

Reply:
Ok, then please remove that change, because a) not needed, b) not related, and c) potentially confusing.

Reply:
Done.